common/utils_test.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 func TestAccumulateRewards(t *testing.T) {
+	t.Skip("skip this test because of constantinople postpone")
```

Comment:
i recommend you to comment out [L67~L90](https://github.com/getamis/eth-indexer/pull/125/commits/d1e46528cef5a57e361740b1fa8b8df2980fb71f#diff-218e81e04c4b21c14fd3343155950a20R67). Keep the origin test for byzantium